### PR TITLE
2.b.iii.2

### PR DESCRIPTION
**Pull request by georgiavram**
Created the RecyclerView and populated with mock data

### CODE REVIEW

File: app/src/main/java/com/avramgeorgiana/newsreader/feature/newslist/adapter/NewsAdapter.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    public void setItems(List<ArticleItemViewModel> news, NewsHandler handler) {
+        this.handler = handler;
+        this.articleList = news;
+        notifyDataSetChanged();
+    }
+
+    public class NewsViewHolder extends RecyclerView.ViewHolder {
+        public final ItemBinding binding;
+        public TextView title;
```
Comment (Misca) [inline app/src/main/java/com/avramgeorgiana/newsreader/feature/newslist/adapter/NewsAdapter.java]: We don't need to keep references to views, binding class is already doing this.

File: app/src/main/java/com/avramgeorgiana/newsreader/view/bindings/ImageBinding.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    public static void setImageUrl(ImageView imageView, Bitmap bitmap) {
+        Glide
+                .with(imageView.getContext())
+                .load(bitmap)
+                .centerCrop()
+                .into(imageView);
+    }
+
+
+//    @BindingAdapter({"imgUrl", "placeholder"})
```
Comment (Misca) [inline app/src/main/java/com/avramgeorgiana/newsreader/view/bindings/ImageBinding.java]: Let's cleanup commented code

**Review by Misca — CHANGES_REQUESTED** [review]
some cleanup.

File: app/src/main/java/com/avramgeorgiana/newsreader/view/bindings/ImageBinding.java (hunk 2 above)
Comment (georgiavram) [reply app/src/main/java/com/avramgeorgiana/newsreader/view/bindings/ImageBinding.java]: Got it. I removed what was unnecessary.

File: app/src/main/java/com/avramgeorgiana/newsreader/feature/newslist/adapter/NewsAdapter.java (hunk 1 above)
Comment (georgiavram) [reply app/src/main/java/com/avramgeorgiana/newsreader/feature/newslist/adapter/NewsAdapter.java]: Removed them. Just leftovers from playing around.

**Review by Misca — APPROVED** [review]
Summer!

**Pull request merged**